Update api-management-faq.md

jboeshart opened the pull request:
Changed the documentation around how to use self-signed certificates so that it uses the PowerShell cmdlet instead of a direct call to the API. Much easier.

azurecla commented:
Hi __@micahtan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jboeshart committed:
Update api-management-faq.md

Changed the documentation around how to use self-signed certificates so that it uses the PowerShell cmdlet instead of a direct call to the API. Much easier.

jboeshart committed:
Update api-management-faq.md

Addition of the New-AzureRmApiManagementBackend command as well.

jboeshart committed:
Update api-management-faq.md

Added the API direct steps back in to the documentation.

jboeshart committed:
Update api-management-faq.md

Added the explicit PowerShell commands to disable certificate chain validation
